src/test/java/com/epam/tc/hw2/ex1/ExerciseOne.java

```diff
@@ -0,0 +1,92 @@
+package com.epam.tc.hw2.ex1;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class ExerciseOne {
+
+    private WebDriver driver;
+
+    @BeforeClass
+    void setupTest() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+    }
+
+    @AfterClass
+    void teardown() {
+        driver.quit(); //12. Close Browser
+    }
+
+    @Test
+    public void test() {
+        SoftAssert softAssert = new SoftAssert();
+
+        driver.get("https://jdi-testing.github.io/jdi-light/index.html"); //1. Open test site by URL
+        String getActualTitle = driver.getTitle();
+
+        softAssert.assertEquals(getActualTitle, "Home Page"); //2. Assert Browser title
+
+        WebElement login = driver.findElement(By.className("navbar-right"))
+                .findElement(By.tagName("li")).findElement(By.tagName("a")); //3. Perform login
+        login.click();
+
+        WebElement name = driver.findElement(By.id("name"));
+        name.click();
+        name.sendKeys("Roman");
+
+        WebElement pass = driver.findElement(By.id("password"));
+        pass.click();
+        pass.sendKeys("Jdi1234");
+
+        WebElement enter = driver.findElement(By.className("fa-sign-in"));
+        enter.click();
+
+        WebElement username = driver.findElement(By.id("user-name")); //4. Assert Username is loggined
+        softAssert.assertEquals(username.getText(), "ROMAN IOVLEV");
+
+        //5. Assert that there are 4 items on the header section are displayed, and they have proper texts
+        List<WebElement> tabs = driver
+                .findElements(By.cssSelector(".m-l8 > li"));
+        softAssert.assertEquals(tabs.stream().map(x -> x.getText()).toArray(),
+                new String[] {"HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS"});
+
+        //6. Assert that there are 4 images on the Index Page, and they are displayed
+        List<WebElement> pict = driver
+                .findElements(By.className("icons-benefit"));
+        int count = pict.size();
+        softAssert.assertEquals(count, 4);
+
+        //7. Assert that there are 4 texts on the Index Page under icons, and they have proper text
+        List<WebElement> pictText = driver
+                .findElements(By.className("benefit-txt"));
+        softAssert.assertEquals(pictText.stream().filter(x -> x.getText() != null).count(), 4);
```

Comment:
Can be simplified if you receive right List of WebElements and you can just check size()